stdlib/public/Concurrency/AsyncStream.swift

```diff
@@ -141,22 +228,48 @@ public struct AsyncStream<Element> {
 
   let produce: () async -> Element?
 
-  /// Construct a AsyncStream buffering given an Element type.
-  ///
-  /// - Parameter elementType: The type the AsyncStream will produce.
-  /// - Parameter maxBufferedElements: The maximum number of elements to
-  ///   hold in the buffer past any checks for continuations being resumed.
-  /// - Parameter build: The work associated with yielding values to the 
-  ///   AsyncStream.
-  ///
-  /// The maximum number of pending elements limited by dropping the oldest
-  /// value when a new value comes in if the buffer would excede the limit
-  /// placed upon it. By default this limit is unlimited.
-  ///
-  /// The build closure passes in a Continuation which can be used in
-  /// concurrent contexts. It is thread safe to send and finish; all calls are
-  /// to the continuation are serialized, however calling this from multiple
-  /// concurrent contexts could result in out of order delivery.
+  /// Constructs an asynchronous stream for an element type, using the
+  /// specified buffering policy and element-producing closure.
+  ///
+  /// - Parameter elementType: The type of element the `AsyncStream`
+  ///   produces.
+  /// - Parameter bufferingPolicy: A `Continuation.BufferingPolicy` value to
+  ///   set the stream's buffering behavior. By default, the stream buffers an
+  ///   unlimited number of elements. You can also set the policy to buffer a
+  ///   specified number of oldest or newest elements.
+  /// - Parameter build: A custom closure that yields values to the
+  ///   `AsyncStream`. This closure receives a `AsyncStream.Continuation`
+  ///   instance that it uses to provide elements to the stream and terminate the
+  ///   stream when finished.
```

Comment:
There's no way to do a test build of documentation content in the Swift standard library. It has to go through the entire build process before we can see the result.  I think this has one extra space, since the continuation lines of each list don't line up with the starting line.  That is, I'd expect this:

```
- first line
  second line
  - nested item
    nested item continued
```